charts/tempo-distributed/templates/_helpers.tpl

```diff
@@ -120,3 +120,14 @@ Return if ingress supports pathType.
 {{- define "tempo.ingress.supportsPathType" -}}
   {{- or (eq (include "tempo.ingress.isStable" .) "true") (and (eq (include "tempo.ingress.apiVersion" .) "networking.k8s.io/v1beta1") (semverCompare ">= 1.18-0" .Capabilities.KubeVersion.Version)) -}}
 {{- end -}}
+
+{{/*
+Return the appropriate apiVersion for PodDisruptionBudget.
+*/}}
+{{- define "tempo.pdb.apiVersion" -}}
+  {{- if and (.Capabilities.APIVersions.Has "policy/v1") (semverCompare ">=1.21-0" .Capabilities.KubeVersion.Version) -}}
```

Comment:
Actually there are two common fixes for this:

1. Stick on what Kubernetes version, which is how loki / loki-simple-distributed stick on.
```suggestion
  {{- if semverCompare ">=1.21-0" .Capabilities.KubeVersion.Version -}}
```

2. Stick on which API available, which is how grafana and prometheus related charts stick on.
```suggestion
  {{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" -}}
```

Reply:
What's the preference? Personally I would have just check the API version but happy to update either way.